moztelemetry/dataset.py

```diff
@@ -154,6 +156,18 @@ def _apply_selection(self, json_obj):
         return dict((name, path.search(json_obj))
                     for name, path in self.selection_compiled.items())
 
+    def _sanitize_dim(self, v):
+        """For String conditions, we should pre-sanitize so that users of
```

Comment:
Could you format this docstring to follow the pep [257 conventions](https://www.python.org/dev/peps/pep-0257/#id17)?